### PR TITLE
Minimal Twitter 5 (ALPHA)

### LINKED ISSUE #116: Option to remove Typefully button
Hello,

I just want to say that this extension literally makes Twitter usable (I do not say this as an understatement). I'd love to support your work in any way I can and understand if the Typefully integration is the best way for you to get that support, but in the spirit of "minimal" it would be great to be able to hide that button via the settings.

Either way, thank you for making such a useful extension!

### LINKED ISSUE #121: Toggle for Communities & Top Articles Button (Twitter Blue)
- Under Navigation section, add toggles for these 2 options: Communities (https://twitter.com/i/communities), Top Articles (https://twitter.com/i/articles)
- Merge PR https://github.com/thomaswang/minimal-twitter/pull/108 first

### LINKED ISSUE #128: Show Labels bugs
Thanx 4 almost the best Twitter extension, I think latest updates cant's let labels disappear & only show on hover. Pleaze fix it.

### LINKED ISSUE #130: Add "Show Less" buttons in toolbar button menu + replace `Customize` in toolbar button tooltip
### Discussed in https://github.com/typefully/minimal-twitter/discussions/104

<div type='discussions-op-text'>

<sup>Originally posted by **darkred** June  8, 2022</sup>
Greetings

I have two suggestions, please:
*(screenshots taken with Chrome 102)*

- In toolbar button menu, when the user expands the 'Navigation' and 'Extras' sections, to have "Show Less" buttons to be able to collapse them:

- In the toolbar button tooltip, replace the current generic `Customize` with the extension name, `Minimal Theme for Twitter`:

Thanks for offering such a useful extension!</div>

### PR DESCRIPTION
**Pull request by thomaswang**
Working on a major version update!

- Fixes #128 
- Fixes #130 
- Closes #116 
- Closes #121 
- Closes #108

### CODE REVIEW

**Comment by thomaswang** [comment]
localhost_3000_ (1)

File: content-scripts/src/modules/initialize.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      saveCurrentReplyToLink();
+      addTypefullyReplyPlug();
+      checkUrlForFollow();
+      checkHomeTimeline();
+      addWriterModeButton();
+      addListsButton();
+      setTimeout(() => {
+        addGrowButton();
+      });
+      extractColorsAsRootVars();
```
Comment (linuz90) [inline content-scripts/src/modules/initialize.js:96]: @thomaswang can you move it to the first place? I think it's better to extract colors before injecting UI

Comment (linuz90) [reply content-scripts/src/modules/initialize.js:96]: I'm referring to `extractColorsAsRootVars`

File: content-scripts/src/modules/options/all.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  changeTimelineBorders,
+  changeTimelineWidth,
+  changeTweetBorders,
+} from "./timeline";
+import { changeGrowButton, changeTypefullyComposerButtons } from "./typefully";
+import { changeWriterMode } from "./writer-mode";
+
+// Array of user preferences, passed to `injectAllChanges`
+export const userPreferences = [
+  "timelineWidth", // 1
```
Comment (linuz90) [inline content-scripts/src/modules/options/all.js:38]: @thomaswang  Can you remove all these `//` comments? If these don't need to be ordered, we don't need them anyway

File: content-scripts/src/modules/initialize.js (hunk 1 above)
Comment (thomaswang) [reply content-scripts/src/modules/initialize.js:96]: 👍 add comment to make sure we keep it at the top

**Pull request merged**